public/schema/JobTemplate.yaml

```diff
@@ -40,6 +40,13 @@ patternProperties:
                             type: string
                           priority:
                             type: number
+                          parameter:
```

Comment:
I wasn't sure but the plural is parameters: https://en.wiktionary.org/wiki/parameter

Reply:
Right - I'll fix that. Or maybe just call it settings for consistency with everything else? Internally I also called the table JobTemplateSettings to be consistent with the other *Settings tables.

Reply:
Go for it - I don't think it makes a difference.